README.md

```diff
@@ -494,3 +494,23 @@ covellipse([0,2], [2 1; 1 4], n_std=2, aspect_ratio=1, label="cov1")
 covellipse!([1,0], [1 -0.5; -0.5 3], showaxes=true, label="cov2")
 ```
 ![covariance ellipses](https://user-images.githubusercontent.com/4170948/84170978-f0c2f380-aa82-11ea-95de-ce2fe14e16ec.png)
+
+## Coefficient plot
+
+Coefficients of a RegressionModel can be plotted with their standard deviation to easily visualize effect size.
```

Comment:
```suggestion
Coefficients of a `RegressionModel` can be plotted with their standard deviation to easily visualize effect size.
```

Reply:
Can you also prefix `RegressionModel` with either the package that defines the type (`StatsAPI`) or the package that implements the code that returns the type?